### DISCUSSION TITLE
Can we handled reset Button pressed

### DISCUSSION THREAD

**Discussion by pallardino** [easepick/easepick#60] [Q&A] [answered]
Hi, is it possible to track when the reset button of amp-plugin is pressed ? I would like to reset my chart data with no date range selected, when reset button is pressed. 

Thanks

**Comment by wakirin** [comment, marked as answer]
You can add a listener for click event and check that clicked element is a reset button.
https://jsfiddle.net/waki/txwrmL7h/

**Reply by pallardino** [reply]
Thanks a lot 👍
